interpreter/cling/lib/Interpreter/CIFactory.cpp

```diff
@@ -657,8 +657,17 @@ static void stringifyPreprocSetting(PreprocessorOptions& PPOpts,
       P.resize(ExeIncl.size());
       // Get foo/include
       llvm::sys::path::append(P, "include");
-      if (llvm::sys::fs::is_directory(P.str()))
+      if (llvm::sys::fs::is_directory(P.str())) {
         utils::AddIncludePaths(P.str(), HOpts, nullptr);
+        llvm::sys::path::append(P, "clang");
+        if (!llvm::sys::fs::is_directory(P.str())) {
```

Comment:
What does this have to do with clad? Can this be moved into a separate PR?

Reply:
This issue was discovered by running `CLING=.../cling ... -Xclang -load clad.so make check-cling`. IMO it is (remotely?) related to the current work.